NEWS.md

```diff
@@ -1,3 +1,7 @@
+ingredients 0.3.9
+---------------------------------------------------------------
+* change `only_numerical` parameter to `variable_type`
```

Comment:
I would add 'in functions aggregated_profiles(), cluster_profiles(), plot() and others, as requested in the [#15](https://github.com/ModelOriented/ingredients/issues/15)'